src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write code here
+const fields = document.querySelectorAll('.field');
+
+[...fields].map(field => {
+  const input = field.children[0];
+
+  const text = input.name.split('').map((char, i) => {
+    if (i === 0) {
+      return char.toUpperCase();
+    }
+
+    if (i !== 0 && char.toUpperCase() === char) {
```

Comment:
it's extra, bc you already have check in 9 line

```suggestion
    if (char.toUpperCase() === char) {
```